src/Screens/Search.tsx

```diff
@@ -33,4 +51,4 @@ const styles = StyleSheet.create({
   },
 });
 
-export default PokemonIndex;
+export default Search;
```

Comment:
Why is this screen called Search now?

Reply:
Overall currently we have to index screens one is with "liked" pokemon and the second one is with All and is used to search a new Pokemon to be liked

Reply:
renamed to "All"